skrub/_dataframe/_common.py

```diff
@@ -47,6 +47,7 @@
     # Inspecting dtypes and casting
     #
     "dtype",
+    "dtypes",
```

Comment:
I don't usually love having names that are so close to each other, but in that case I guess it's ok because those are the names used by polars & pandas, and the arguments passed to `specialize` will ensure we get an exception in case of typos...

Reply:
BTW I wonder if the default implementation of generic functions should raise a `TypeError` rather than `NotImplementedError` -- to be discussed in another issue